From dc74b6646bc45c7ed28d8d339f0edbe21c2a38ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Fri, 19 Oct 2018 08:02:12 +0200
Subject: [PATCH] fix default scheme in custom partitioning

Update to LVM Thin Provisioning there too.

Fixes QubesOS/qubes-issues#3225
---
 pyanaconda/core/constants.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/core/constants.py b/pyanaconda/core/constants.py
index 161e5575a..41b7b613e 100644
--- a/pyanaconda/core/constants.py
+++ b/pyanaconda/core/constants.py
@@ -29,7 +29,7 @@ SELINUX_DEFAULT = -1
 # where to look for 3rd party addons
 ADDON_PATHS = ["/usr/share/anaconda/addons"]
 
-from pykickstart.constants import AUTOPART_TYPE_LVM
+from pykickstart.constants import AUTOPART_TYPE_LVM_THINP
 
 # common string needs to be easy to change
 from pyanaconda import product
@@ -236,7 +236,7 @@ CMDLINE_LIST = ["addrepo"]
 
 # The default autopart type is lvm.
 # FIXME: Move this constant to the storage module.
-DEFAULT_AUTOPART_TYPE = AUTOPART_TYPE_LVM
+DEFAULT_AUTOPART_TYPE = AUTOPART_TYPE_LVM_THINP
 
 # Is the default autopart type selected?
 AUTOPART_TYPE_DEFAULT = -1
-- 
2.17.2

